tests/test_utils/test_timer.py

```diff
@@ -0,0 +1,46 @@
+# tests.test_utils.test_timer
+# Tests for the stand alone timer functions in Yellowbrick utils.
+#
+# Author:   ZJ Poh <poh.zijie@gmail.com>
+# Created:  Tue Jul 17 21:11:11 2018 -0700
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+"""
+Tests for the stand alone timer functions in Yellowbrick utils.
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import unittest
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+from yellowbrick.utils.timer import *
+
+##########################################################################
+## Helper Function Tests
+##########################################################################
+
+class TestTimer(unittest.TestCase):
+    """
+    Timer functions and utilities
+    """
+
+    @mock.patch('time.time', mock.MagicMock(return_value=1234.))
```

Comment:
Patching time is an excellent method of testing the timer! If you change the mock to have a `side_effect` instead of a `return_value` that is a sequence, e.g. `mock.Mock(side_effect=[1234.2, 1242.8])` then the mock will return the values in order of calls. You can then use this to test the `Timer` difference of 8.6 seconds.

Reply:
Thanks for pointing that out!